modules/hosted-control-planes-overview.adoc

```diff
@@ -8,7 +8,7 @@
 [id="hosted-control-planes-overview_{context}"]
 = Introduction to {hcp}
 
-{hcp-capital} is available by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#cluster_mce_overview[{mce} version 2.0 or later] on the following platforms:
+{hcp-capital} is available by using the link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html/clusters/cluster_mce_overview#cluster_mce_overview[{mce} version 2.0 or later] on the following platforms:
```

Comment:
Here we say "A supported version" of MCE. I would take version out of here; I think we have even EOL'ed 2.2 and earlier.

Reply:
I removed the version-specific language. Thanks!